tests/testthat/test-instance_api.R

```diff
@@ -0,0 +1,105 @@
+skip_if_offline()
+
+broken_instance_settings <- function() {
+  InstanceSettings$new(
+    list(
+      owner = "foo",
+      name = "bar",
+      id = "...",
+      schema_str = "foo,bar",
+      schema_id = "...",
+      git_repo = "...",
+      keep_artifacts_local = TRUE,
+      api_url = "https://foo.lamin.ai"
+    )
+  )
+}
+
+test_that("test get_schema", {
```

Comment:
Definitely not important but I think you are supposed to read this as a sentence starting with "Test that..." so the extra "test" is a bit redundant.

Reply:
fixed!